addons/website_event_exhibitor/static/src/js/event_exhibitor.js

```diff
@@ -0,0 +1,244 @@
+odoo.define('website_event_exhibitor.event_exhibitor', function (require) {
```

Comment:
After discussing this a bit with the team, we decided to create a separate task for that feature.
It should be assigned to you already.

Indeed, this raises many questions and we would like to be able to experiment with this without slowing down the other important points of this PR.

I suggest you cherry-pick this commit into another branch to keep the code and then remove it from this PR.

If you have questions about that, don't hesitate to ping me on Discord!

Thanks :)

Reply:
Ok, I created a new branch from master and I added the commit. It will be removed from the PR.